data.json

```
@@ -1012,5 +1012,11 @@
     "githubUsername": "tips367",
     "imageUrl": "https://user-images.githubusercontent.com/41302067/123315141-03eb1900-d549-11eb-9021-a4c3b985c95d.png",
     "issueNumber": 562
-  }
+  },
+  {
+    "name": "Jayesh Srivastava",
+    "githubUsername": "jayesh-srivastava",
+    "imageUrl": "https://user-images.githubusercontent.com/49337716/124381232-eeab8280-dcde-11eb-84ed-c54a5bfcf208.png",
+    "issueNumber": 566
+}
```

Align this with the other brackets

```suggestion
  }
```